modules/ROOT/partials/configuration/text_color.adoc

```diff
@@ -49,6 +107,122 @@ tinymce.init({
 });
 ----
 
+If `+color_map+` is not explicitly set, {productname} populates the foreground and background text color grids with a default set of 22 colors.
+
+==== Example: explicitly setting `+color_map+` to the default {productname} color set
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  toolbar: 'forecolor backcolor',
+  color_map: [
+    '#BFEDD2', 'Light Green',
+    '#FBEEB8', 'Light Yellow',
+    '#F8CAC6', 'Light Red',
+    '#ECCAFA', 'Light Purple',
+    '#C2E0F4', 'Light Blue',
+    '#2DC26B', 'Green',
+    '#F1C40F', 'Yellow',
+    '#E03E2D', 'Red',
+    '#B96AD9', 'Purple',
+    '#3598DB', 'Blue',
+    '#169179', 'Dark Turquoise',
+    '#E67E23', 'Orange',
+    '#BA372A', 'Dark Red',
+    '#843FA1', 'Dark Purple',
+    '#236FA1', 'Dark Blue',
+    '#ECF0F1', 'Light Gray',
+    '#CED4D9', 'Medium Gray',
+    '#95A5A6', 'Gray',
+    '#7E8C8D', 'Dark Gray',
+    '#34495E', 'Navy Blue',
+    '#000000', 'Black',
+    '#ffffff', 'White'
+  ]
+});
+----
+
+
+[[color_map_background]]
+=== `color_map_background`
+
+This option allows for specifying a map of the text colors that will appear in the background text color grid.
+
+If it is not set, the background text color grid takes it values from the `+color_map+` array.
+
+And, if the `+color_map+` array is, further, not set, the background text color grid takes it values from the {productname} default color set.
+
+That is, the background text color grid takes its values in the following priority order:
+
+. a set `color_map_background` color array is used in preference to
+. a set `color_map` color array, which is used in preference to
+. the default {productname} color array.
+
+*Type:* `+Array+`
+
+==== Example: using `color_map_background`
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your html
+  toolbar: 'forecolor backcolor',
+  color_map_background: [
+    '000000', 'Black',
+    '808080', 'Gray',
+    'FFFFFF', 'White',
+    'FF0000', 'Red',
+    'FFFF00', 'Yellow',
+    '008000', 'Green',
+    '0000FF', 'Blue'
+  ]
+});
+----
+
+When end-users add a further custom color to the background text color grid, that color is added to the background text color grid and stored in the `color_map_background` palette.
+
+NOTE: When a user adds a custom color to the `color_map_background` palette, the editor does not replicate the new color in the `color_map_foreground` palette.
```

Comment:
Have you tested this? Seems like something only in local storage, not within the option value. It might appear in both if a user selects custom colors for later use.

Reply:
To quote my commit message:

> After further testing, re-wrote notes about end-user customisation and placed re-write into general IMPORTANT admonition set after all three color map array sections.